.github/workflows/ci.yml

```diff
@@ -41,4 +51,29 @@ jobs:
         with:
           type: ${{ job.status }}
           job_name: Build
-          url: ${{ secrets.SLACK_WEBHOOK }}
\ No newline at end of file
+          url: ${{ secrets.SLACK_WEBHOOK }}
+
+  publish:
+    name: Publish Artifacts
+    needs: [test]
+    if: github.event_name != 'pull_request' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v'))
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: coursier/cache-action@v6
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 17
+          cache: "sbt"
+
+      - name: Publish artifacts
+        env:
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USER }}
```

Comment:
Do we need some special configuration for this repository? I don't know if it's necessary to add secrets to each repo manually

Reply:
I don't have access to repo and organization secrets, but if it `SONATYPE_USER` is an organization one, then adding it it's needed only once. Given fact that there are already 2 libraries which are using it, I assumed that this is already organization secret. Ca you check that? Also calling @t3hnar